package.json

```diff
@@ -23,7 +23,7 @@
   "author": "Microsoft Corporation",
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.10.1",
```

Comment:
You'll need to bump the version to `0.1.2-0`, right?
`

Reply:
We can